### ISSUE TITLE
stuck at show-full-ui

### ISSUE THREAD

**Issue by Ebraama06** [waydroid/waydroid#334]
So every time I run "waydroid show-full-ui"
it output:
 Starting waydroid session
 [gbinder] Service manager /dev/anbox-binder has appeared
and then the pc slowdown like loading but but after 30min nothing and sometimes after so work around it, lineageos loading screen show but stuck it  

 - Waydroid tools Version: 1.2.0-2
 - Waydroid Images Version: dev 18.1_202111291420-1 from aur

 - OS: arch x86_64
 - CPU: AMD® A6-5400b 
 - GPU: AMD Radeon HD 7540D 
 - Kernel version: linux-zen 5.16.15.zen1-1
 - Host mesa version: 21.3.7-2
 - Desktop: Gnome 41.3

 - Output of `waydroid log`:
 (000312) [19:49:39] session manager stopped, stopping container and waiting...
(000312) [19:49:39] % lxc-stop -P /var/lib/waydroid/lxc -n waydroid -k
lxc-start: waydroid: conf.c: run_buffer: 321 Script exited with status 126
lxc-start: waydroid: start.c: lxc_end: 988 Failed to run lxc.hook.post-stop for container "waydroid"
(000312) [19:49:39] % /usr/lib/waydroid/data/scripts/waydroid-net.sh stop
(000312) [19:49:39] % umount /var/lib/waydroid/rootfs/vendor/waydroid.prop
(000312) [19:49:39] % umount /var/lib/waydroid/rootfs/vendor
(000312) [19:49:39] % umount /var/lib/waydroid/rootfs
(000312) [19:49:39] % umount /var/lib/waydroid/data
(000312) [19:49:39] % chmod 666 -R /dev/anbox-binder
(000312) [19:49:39] % chmod 666 -R /dev/anbox-vndbinder
(000312) [19:49:39] % chmod 666 -R /dev/anbox-hwbinder
(000312) [19:49:39] Container manager is waiting for session to load
(001428) [19:49:52] WayDroid container is STOPPED
(001430) [19:49:52] WayDroid session is not started
(001431) [19:49:52] % chmod 666 -R /dev/anbox-binder
(001431) [19:49:52] % chmod 666 -R /dev/anbox-vndbinder
(001431) [19:49:52] % chmod 666 -R /dev/anbox-hwbinder
(001431) [19:49:52] Container manager is waiting for session to load
(001438) [19:50:08] Save config: /var/lib/waydroid/waydroid.cfg
(001438) [19:50:08] % lxc-info --version
4.0.12
(001438) [19:50:08] % mkdir -p /var/lib/waydroid/lxc/waydroid
(001438) [19:50:08] % cp -fpr /usr/lib/waydroid/data/configs/config_2 /var/lib/waydroid/lxc/waydroid/config
(001438) [19:50:08] % sed -i s/LXCARCH/x86_64/ /var/lib/waydroid/lxc/waydroid/config
(001438) [19:50:08] % mv /var/lib/waydroid/config_nodes /var/lib/waydroid/lxc/waydroid
(001445) [19:50:11] Starting waydroid session
(001445) [19:50:11] Save session config: /var/lib/waydroid/session.cfg
(001445) [19:50:11] UserMonitor service is not even started
(001445) [19:50:11] Clipboard service is not even started
(001431) [19:50:11] % /usr/lib/waydroid/data/scripts/waydroid-net.sh start
(001431) [19:50:11] % mount /usr/share/waydroid-extra/images/system.img /var/lib/waydroid/rootfs
(001431) [19:50:11] % mount -o remount,ro /usr/share/waydroid-extra/images/system.img /var/lib/waydroid/rootfs
(001431) [19:50:11] % mount /usr/share/waydroid-extra/images/vendor.img /var/lib/waydroid/rootfs/vendor
(001431) [19:50:12] % mount -o remount,ro /usr/share/waydroid-extra/images/vendor.img /var/lib/waydroid/rootfs/vendor
(001431) [19:50:12] % mount -o bind /var/lib/waydroid/waydroid.prop /var/lib/waydroid/rootfs/vendor/waydroid.prop
(001431) [19:50:12] % mount -o bind /home/ebraam/.local/share/waydroid/data /var/lib/waydroid/data
(001431) [19:50:12] % umount -l /sys/fs/cgroup/schedtune
umount: /sys/fs/cgroup/schedtune: no mount point specified.
(001431) [19:50:12] % chmod 777 -R /dev/ashmem
(001431) [19:50:12] % chmod 777 -R /dev/dri
(001431) [19:50:12] % chmod 777 -R /dev/fb0
(001431) [19:50:12] % lxc-start -P /var/lib/waydroid/lxc -F -n waydroid -- /init
(001431) [19:50:12] New background process: pid=1503, output=background
(001431) [19:50:12] Save session config: /var/lib/waydroid/session.cfg
(001431) [20:14:26] session manager stopped, stopping container and waiting...
(001431) [20:14:26] % lxc-stop -P /var/lib/waydroid/lxc -n waydroid -k
lxc-start: waydroid: conf.c: run_buffer: 321 Script exited with status 126
lxc-start: waydroid: start.c: lxc_end: 988 Failed to run lxc.hook.post-stop for container "waydroid"
(001431) [20:14:28] % /usr/lib/waydroid/data/scripts/waydroid-net.sh stop
(001431) [20:14:31] % umount /var/lib/waydroid/rootfs/vendor/waydroid.prop
(001431) [20:14:31] % umount /var/lib/waydroid/rootfs/vendor
(001431) [20:14:31] % umount /var/lib/waydroid/rootfs
(001431) [20:14:31] % umount /var/lib/waydroid/data
(001431) [20:14:31] % chmod 666 -R /dev/anbox-binder
(001431) [20:14:31] % chmod 666 -R /dev/anbox-vndbinder
(001431) [20:14:31] % chmod 666 -R /dev/anbox-hwbinder
(001431) [20:14:31] Container manager is waiting for session to load
(075032) [20:27:41] % tail -n 60 -F /var/lib/waydroid/waydroid.log
(075032) [20:27:41] *** output passed to waydroid stdout, not to this log ***

and thank you

**Comment by mizzunet** [comment]
Are you having bootloop?

**Comment by Ebraama06** [comment]
> 

the only time waydroid opened a window yes 
lineageos just give me loading

**Comment by aleasto** [comment]
> Waydroid Images Version: dev 18.1_202111291420-1 from aur

This is not officially supported

**Comment by AndrewPrzelucki** [comment]
I updated waydroid yesterday and to my dismay it wouldn't start anymore! It was stuck on the boot logo every time - this never happened before. I uninstalled and installed back (clean install) waydroid but that didn't help. I run waydroid on Kubuntu in an X session, through Weston. The solution right now (for me) is to start the waydroid session not with "waydroid session start" and "waydroid show-full-ui" but only with "WAYLAND_DISPLAY=/run/user/1000/wayland-1 waydroid first-launch". I successfully reinstalled waydroid with Gapps and got back my whole install - including game saves! I didn't do anything with the binder thing or anything else, just running Weston and then waydroid first-launch every time. Works!
